spiped/main.c

```diff
@@ -27,6 +28,12 @@ usage(void)
 	exit(1);
 }
 
+static void
+signal_handler(int signo)
```

Comment:
I'd rename this to "diediedie_handler" to make the intention clearer.  "signal_handler" is a bit vague...